apps/csm-portal/backend/cmd/server/main_test.go

```
@@ -0,0 +1,117 @@
+	}{
+		{name: "valid https base URL, no public URL", baseURL: "https://sftpgo.internal.example.com", wantExitNonZero: false},
+		{name: "valid https for both", baseURL: "https://sftpgo.internal.example.com", publicBaseURL: "https://share.example.com", wantExitNonZero: false},
+		{name: "http base URL rejected", baseURL: "http://sftpgo.internal.example.com", wantExitNonZero: true},
+		{name: "http public base URL rejected", baseURL: "https://sftpgo.internal.example.com", publicBaseURL: "http://share.example.com", wantExitNonZero: true},
+		{name: "userinfo in base URL rejected", baseURL: "https://user:pass@sftpgo.internal.example.com", wantExitNonZero: true},
+		{name: "userinfo in public base URL rejected", baseURL: "https://sftpgo.internal.example.com", publicBaseURL: "https://user:pass@share.example.com", wantExitNonZero: true},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			cmd := exec.Command(os.Args[0], "-test.run=^TestLoadSftpgoConfigRejectsBadURLs$")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)